build/builder.sh

```diff
@@ -6,7 +6,7 @@ image="cockroachdb/builder"
 
 function init() {
   docker build --tag="${image}" - <<EOF
-FROM golang:1.4.2
+FROM golang:1.5
```

Comment:
FYI, in order to get circleci to use this you'll have to push a new builder image and update the `tag` in `build/circle-deps.sh`.